pypi_on_github_indexer/__main__.py

```diff
@@ -97,17 +100,21 @@ def main():
         index_file = Path(index_dir) / index_file
         parser = IndexHTMLParser()
         links_data = parser.get_index_data(str(index_file))
-
+        url = ("git+%(repo_url)s@%(repo_tag)s#egg=%(package_name)s-%(package_version)s" %
+               dict(repo_url=args.repo_url,
+                    repo_tag=args.repo_tag,
+                    package_version=str(package_version),
+                    package_name=package_name))
+        if args.repo_commit is not None:
+            url = "%s@%s" % (url, args.repo_commit)
+        if args.repo_subdirectory is not None:
+            url = "%s#subdirectory=%s" % (url, args.repo_subdirectory)
```

Comment:
Have you tried this? From https://pip.pypa.io/en/stable/reference/pip_install parameters in a url should be separated with `&`
```
pip install -e vcs+protocol://repo_url/#egg=pkg&subdirectory=pkg_dir
```
That would give us something like
```python
url = "%s&subdirectory=%s" % (url, args.repo_subdirectory)

Reply:
OMG. I've been doing this wrong the whole time. Already updated